doc/api/console.markdown

```diff
@@ -72,18 +72,18 @@ object. This is useful for inspecting large complicated objects. Defaults to
 - `colors` - if `true`, then the output will be styled with ANSI color codes.
 Defaults to `false`. Colors are customizable, see below.
 
-### console.time(label)
+### console.time(timerName)
 
-Used to calculate the duration of a specific operation. To start a timer, call
-the `console.time()` method, giving it a name as only parameter. To stop the
-timer, and to get the elapsed time in milliseconds, just call the
-[`console.timeEnd()`](#console_console_timeend_label) method, again passing the
-timer's name as the parameter.
+Starts a timer that can be used to compute the duration of an operation. Timers
+are identified by a unique name. Use the same name when you call
+[`console.timeEnd()`](#console_console_timeend_timername) to stop the timer and
+output the elapsed time in milliseconds. Timer durations are accurate to the
+sub-millisecond.
```

Comment:
Nittiest of all possible nits: The last sentence describes the timer precision, not the accuracy.

Slightly less nitty nit: `sub-millisecond` is true, but we know that it will be three decimal places, so you could just say it's to the microsecond.

Of course, I can't come up with wording for either of these nits that's better than what's already there, so ¯\\_(ツ)_/¯